doc/api/prev_api_changes/api_changes_3.6.0/behaviour.rst

```diff
@@ -0,0 +1,269 @@
+:rc:`backend` anymore.
+
+Default ``rcParams["animation.convert_args"]`` changed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It now defaults to ``["-layers", "OptimizePlus"]`` to try to generate smaller
+GIFs. Set it back to an empty list to recover the previous behavior.
+
+Style file encoding now specified to be UTF-8
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It has been impossible to import Matplotlib with a non UTF-8 compatible locale
+encoding because we read the style library at import time. This change is
+formalizing and documenting the status quo so there is no deprecation period.
+
+MacOSX backend uses sRGB instead of GenericRGB color space
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MacOSX backend now display sRGB tagged image instead of GenericRGB which is an
+older (now deprecated) Apple color space. This is the source color space used
+by ColorSync to convert to the current display profile.
+
+Renderer optional for ``get_tightbbox`` and ``get_window_extent``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `.Artist.get_tightbbox` and `.Artist.get_window_extent` methods no longer
+require the *renderer* keyword argument, saving users from having to query it
+from ``fig.canvas.get_renderer``. If the *renderer* keyword argument is not
+supplied, these methods first check if there is a cached renderer from a
+previous draw and use that. If there is no cached renderer, then the methods
+will use ``fig.canvas.get_renderer()`` as a fallback.
+
+``FigureFrameWx`` constructor, subclasses, and ``get_canvas``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``FigureCanvasWx`` constructor gained a *canvas_class* keyword-only
+parameter which specifies the canvas class that should be used. This parameter
+will become required in the future. The ``get_canvas`` method, which was
+previously used to customize canvas creation, is deprecated. The
+``FigureFrameWxAgg`` and ``FigureFrameWxCairo`` subclasses, which overrode
+``get_canvas``, are deprecated.
+
+``FigureFrameWx.sizer``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+... has been removed. The frame layout is no longer based on a sizer, as the
+canvas is now the sole child widget; the toolbar is now a regular toolbar added
+using ``SetToolBar``.
+
+Incompatible layout engines raise
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``tight_layout`` and ``constrained_layout`` are incompatible if a colorbar has
+been added to the figure. Invoking the incompatible layout engine used to warn,
+but now raises with a ``RuntimeError``.
+
+``CallbackRegistry`` raises on unknown signals
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When Matplotlib instantiates a `.CallbackRegistry`, it now limits callbacks to
+the signals that the registry knows about. In practice, this means that calling
+`~.FigureCanvasBase.mpl_connect` with an invalid signal name now raises a
+`ValueError`.
```

Comment:
Markup for builtin exception classes is inconsistent between e.g. here and the next entry.

Reply:
Fixed.